pkg/agent/datapath/cache/svc_index.go

```diff
@@ -0,0 +1,68 @@
+package cache
+
+import (
+	"sync"
+
+	"github.com/contiv/ofnet/ofctrl"
+)
+
+type SvcIndex struct {
+	lock sync.RWMutex
+	// dnatMap key is ip-port-protocol, value is flowID in NatBrDnatTable
+	dnatMap map[string]*ofctrl.Flow
+	// svcMap key is svcNs/svcName
+	svcMap map[string]*SvcOvsInfo
+}
+
+func GenDnatMapKey(ip, protocol string, port int32) string {
+	return ip + "-" + string(port) + "-" + protocol
```

Comment:
`string(port)` not humanreadable, may be `strconv.Itoa(port)` more suitable?

Reply:
done